src/ShallowTraversal.js

```diff
@@ -151,3 +154,21 @@ export function getTextFromNode(node) {
     .join('')
     .replace(/\s+/, ' ');
 }
+
+export const findRefInOwnedChildren = (node, ref, isRootNode) => {
```

Comment:
this being exported, having `isRootNode` be provideable by the consumer seems brittle. Could we split this up into separate functions, so that the recursion implementation details are hidden?